holidayextras/ui-toolkit
# Fix lozenges

#### What does this PR do? (please provide any background)
- The Lozenges aren't written in the same way as other components on the toolkit.
- They don't include sizes, or utilise the colours in UI-Toolkit, they rely on bootstrap being introduced to be styled
- This PR addressed that issue and adds in Toolkit styling.

#### What tests does this PR have?
- I have added in more test to check size classes are being added to the element.

#### How can this be tested?
- Run `grunt docs` is the best place to start
- Once the docs have opened click on `lozenges` in the navigation
- You should see a full set of beautiful lozenges, you can edit to code by clicking on the `code` tab ont eh doc (it looks like you can't edit it but try using backspace.

#### Screenshots / Screencast
<img width="858" alt="screen shot 2015-08-11 at 14 55 34" src="https://cloud.githubusercontent.com/assets/778942/9199203/0ca6453e-4039-11e5-8de8-415adb79d402.png">

#### What gif best describes how you feel about this work?
![adventure time](http://media0.giphy.com/media/8Ry7iAVwKBQpG/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
